pyaedt/edb_core/edb_data/terminals.py

```diff
@@ -476,6 +476,13 @@ class PadstackInstanceTerminal(Terminal):
     def __init__(self, pedb, edb_object):
         super().__init__(pedb, edb_object)
 
+    @property
+    def position(self):
+        edb_padstack_instance = self._edb_object.GetParameters()
+        if edb_padstack_instance[0]:
+            return self._pedb.padstacks.instances[edb_padstack_instance[1].GetId()].position
```

Comment:
I suggest to create a new object 

`return EdbPadstack(edb_padstack_instance[1]).position`

instead of querying the how instances.

Reply:
Yes I agree thanks